### PR TITLE
Allow Symfony YAML 5.0

### PR DESCRIPTION
**Pull request by lookyman**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,16 +7,22 @@ matrix:
       env: YAML=3.0
     - php: '7.1'
       env: YAML=4.0
+    - php: '7.1'
+      env: YAML=5.0
```
Comment (cebe) [inline .travis.yml]: yaml 5.0 seems to require php 7.2 so we can not test this combination.

**Comment by cebe** [comment]
Seem symfony/yaml 4.4 breaks one of the yaml file tests as it does not read them the same as prior versions did...

**Comment by lookyman** [comment]
Yep, looks like it's not as straightforward as I thought.

**Comment by lookyman** [comment]
This is probably the culprit: https://github.com/symfony/yaml/commit/110e3e9eaa7827def84cb1cfee95a6cfde4bf608

I haven't checked it out yet, but it's the only relevant commit message referencing inline YAML since 4.3.0.

**Comment by cebe** [comment]
I have created a test script and reported a bug here: https://github.com/symfony/symfony/issues/34805

**Pull request closed without merging**